CHANGES.md

```diff
@@ -1,6 +1,417 @@
+
+Standard Library Modules
+------------------------
+* added a new 'CTypes' module that defines Chapel aliases for C types  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/CTypes.html)
+* added optional arguments to `set` initializers for more control over resizing  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Set.html#Set.set)
+* added an overload of `map.getValue()` that accepts a sentinel value  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Map.html#Map.map.getValue)
+* changed `map.getValue()` to `throw` rather than halting  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Map.html#Map.map.getValue)
+* enabled casts from integral types to `c_void_ptr`  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/CTypes.html#CTypes.c_void_ptr)
+* enabled `min()`/`max()` routines to be promoted using array arguments
+* added `subprocess.abort()` and `subprocess.alarm()` to 'Subprocess'  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Subprocess.html#Subprocess.subprocess.abort  
+   and https://chapel-lang.org/docs/1.26/modules/standard/Subprocess.html#Subprocess.subprocess.alarm)
+* updated `Math.cproj()` to return a `complex` rather than `real` value  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/Math.html#Math.cproj)
+* added support for a standalone (channel-less) `IO.readline()` routine  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/IO.html#IO.readline)
+* updated `bigint.probablyPrime()` to return a new enum `BigInteger.primality`  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.primality  
+  and https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.probablyPrime)
+* swapped the order of the values returned from `bigint.getD2Exp()`  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/BigInteger.html#BigInteger.bigint.getD2Exp)
+
+Package Modules
+---------------
+* added a new 'CopyAggregation' module supporting aggregated copies  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/CopyAggregation.html)
+* added a new 'Socket' package module to support network programming  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/Socket.html)
+* added a new 'Channel' package module to support communicating between tasks  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/Channel.html)
+* changed the 'TOML' package to sort keys when writing them out  
+  (see https://chapel-lang.org/docs/1.26/modules/packages/TOML.html)
+
+Tool Improvements
+-----------------
+* added a new script for displaying available runtime configurations  
+  (see `$CHPL_HOME/util/chplenv/printchplbuilds.py`)
+
+Performance Optimizations / Improvements
+----------------------------------------
+* significantly improved fine-grained communication on Slingshot-11 networks
+* made promotions with slices more efficient with `-schpl_serializeSlices=true`
```

Comment:
Should this be under dev changes given the non-user facing name?

Reply:
I considered that, but ended up putting it here since we have told users about it from time to time, and I think we'll likely talk about it in the release notes as well.  I view the name as being more an indication of its temporary nature (fingers crossed) than something users shouldn't be aware of or use.